utils/ansible/ftest/README.md

```diff
@@ -0,0 +1,198 @@
+  |  |  |--{daos_source_dir = /home/foo/work/daos}
+  |  |--wolf-999
+  |  |  |--{daos_hugepages_nb = 8182}
+  |  |  |--{daos_runtime_dir = /home/foo/daos}
+  |  |--{daos_hugepages_nb = 8182}
+  |--@ungrouped:
+  |--{daos_runtime_dir = /home/foo/daos}
+```
+
+### Playing Playbook
+
+When the inventory is defined, then the
+[playbook](https://docs.ansible.com/ansible/latest/user_guide/playbooks_intro.html)
+*ftest.yml* could be played thanks to the `ansible-playbook` command.
+
+```bash
+ansible-playbook -i my-inventory.yml ftest.yml
+```
+
+If the play succeed, the different nodes of the platform should be well configured and the two bash
+scripts `daos-launch.sh` and `daos-make.sh` should be available in the DAOS runtime directory
+defined in the inventory. Usage of these two scripts will be detailed in the following sections.
+
+> :bulb: As *ftest.yml* playbook is
+> [idempotent](https://docs.ansible.com/ansible/latest/reference_appendices/glossary.html#term-Idempotency),
+> it could be safely run several times without side effect.
+
+## Installing DAOS Binaries
+
+Building and installing binaries could be done thanks to the generated bash script `daos-make.sh`.
+This last script support several options and the two sub-commands `install` and `update`.
+
+More details on the supported options could be found with running the command with the `--help`
+option.
+
+The `install` sub-command should be used to install/reinstall from scratch all the DAOS binaries and
+their dependencies such as the *spdk* or the *mercury* libraries. For example, the *foo* user could
+used the following command line to install or reinstall the DAOS binaries and its dependencides into
+the `/home/foo/daos/install` directory.
+
+```bash
+/home/foo/daos/daos-make.sh -v -j 32 -f install
+```
+
+When the previous *install* step has been done and the DAOS source code tests have been updated, the
+DAOS binaries could be build and reinstalled thanks to the `update` sub-command.  This last one will
+should be far more quicker than a full reinstall.
+
+```bash
+/home/foo/daos/daos-make.sh -v -j 32 update
+```
+
+> :bulb: The option `-j 32` (i.e. one per core) seems to be a good compromise between compilation
+> reliability and speed.
+
+## Launching Functional Tests
+
+Launching the DAOS functional tests could be done thanks to the generated bash script
+`daos-launch.sh`. This last script supports several options and it is also possible to give some
+options to the original `launch.py` python script of the DAOS functional test plaftrom.
+
+More details on the supported options and the way to passing options to `launch.py` could be found
+with running the command with the `--help` option.
+
+After successfully installing the DAOS binaries and dependencies, the user *foo* could run all the
+test(s) with the tag `hello_world` thanks to the following command line.
+
+
+```bash
+/home/foo/daos/daos-launch.sh -v -- --nvme=auto  hello_world
```

Comment:
Where should we specify the server nodes and the client nodes when running this? Do we support file name in addition to the tag?

Reply:
The list of server nodes are automatically added  from the ansible inventory. The end user do not have to bother with that. More details could be found at line 13-16 of the file _templates/daos-launch.sh.j2_